test/functional/api/v2/job_invocations_controller_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_plugin_helper'
```

Comment:
I meant that syntax of this test file is not in rspec style but that's not a big deal. Let's ignore it for now.